manifests/v/v2raya/win/1.5.8.1/v2raya.win.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: v2raya.win
+PackageVersion: 1.5.8.1
+PackageLocale: en-US
+Publisher: The v2rayA Developer Community

```

This must match control panel exactly
```suggestion
Publisher: The v2rayA developer community
```